### ISSUE TITLE
Toggle for videos and channels to only allow comments from local users

### ISSUE THREAD

**Issue by emansom** [Chocobozzz/PeerTube#5461]
### Describe the problem to be solved

Currently on a regular basis, spambots comment on videos.

### Describe the solution you would like

The ability to only allow comments from local users, configurable per video and channel basis.

**Comment by emansom** [comment]
With a visual indication that this'd break ActivityPub interaction with the audience.

**Comment by Vertux** [comment]
@emansom I agree with you, that would be a very helpful and importtant feature.

I think it would be nice to have two independent options:

- [ ] Turn off/on remote comments

- [ ] Turn off/on local comments
